### DISCUSSION TITLE
How to develop cross platform programs using vb.net in Avalonia?

### DISCUSSION THREAD

**Discussion by rrokoko** [AvaloniaUI/Avalonia#17132] [Q&A] [answered]
I am a VB.net enthusiast and I want to develop cross platform programs using VB.net in Avalonia, but I have not found the relevant method. How can I do it and can you provide assistance? I need more detailed steps. thank you!

**Comment by stevemonaco** [comment, marked as answer]
There's no official support for VB.NET. See these older discussions regarding reasons and potential options:

https://github.com/AvaloniaUI/Avalonia/discussions/10387
https://github.com/AvaloniaUI/Avalonia/discussions/14286
